src/Microsoft.DotNet.Build.Tasks/Targets/packageresolve.targets

```diff
@@ -34,23 +42,73 @@
       $(ResolveAssemblyReferencesDependsOn);
       ResolveNuGetPackages;
     </ResolveAssemblyReferencesDependsOn>
+    <NugetTargetFrameworkMoniker Condition="'$(NugetTargetFrameworkMoniker)' == ''">$(TargetFrameworkMoniker)</NugetTargetFrameworkMoniker>
   </PropertyGroup>
 
   <Target Name="ResolveNuGetPackages" 
           Condition="'$(ResolveNuGetPackages)'=='true'">
 
-    <ResolveNuGetPackages PackagesConfigs="$(ProjectPackagesConfigFile)"
+    <!-- Use old task for packages.config -->
+    <ResolveNuGetPackages Condition="'$(ProjectPackagesConfigFile)' != ''"
+                          PackagesConfigs="$(ProjectPackagesConfigFile)"
                           PackageRoot="$(PackagesDir)"
                           Platform="$(PlatformTarget)"
                           Configuration="$(Configuration)"
                           Language="$(Language)"
-                          TargetFramework="$(TargetFrameworkMoniker)"
+                          TargetFramework="$(NugetTargetFrameworkMoniker)"
                           TargetPlatformMoniker="$(TargetPlatformMoniker)">
 
       <Output TaskParameter="ResolvedAnalyzers" ItemName="Analyzer" />
       <Output TaskParameter="ResolvedReferences" ItemName="Reference" />
       <Output TaskParameter="ResolvedCopyLocal" ItemName="None" />
     </ResolveNuGetPackages>
+
+    <PropertyGroup>
+      <!-- Temporary workaround -->
```

Comment:
It's working around the fact that ResolveNuGetPackageAssets doesn't currently handle PCL profiles.  Additionally, we should probably change the TFM being used by all our projects to something that actually makes sense for what they are targeting.  Probably the new Dev14 PCL TFM (once it exists) I'll file an issue.